adapter/stdLogger/stdLogger_test.go

```diff
@@ -0,0 +1,248 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package stdLogger
+
+import (
+	"errors"
+	"io"
+	"os"
+	"reflect"
+	"strings"
+	"testing"
+	"time"
+
+	"github.com/golang/protobuf/proto"
+	"google.golang.org/genproto/googleapis/rpc/status"
+	"istio.io/mixer/adapter/stdLogger/config"
+	"istio.io/mixer/pkg/aspect"
+	"istio.io/mixer/pkg/aspect/logger"
+	"istio.io/mixer/pkg/registry"
+)
+
+func TestRegister(t *testing.T) {
```

Comment:
Can be replaced with a boilerplate call to the adaptertesting.TestAdapterInvariants call (available as soon as my code coverage PR is approved)